src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
+const table = document.querySelector('.field');
+const tbody = table.querySelector('tbody');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+
+appendRow.addEventListener('click', () => {
+  const createRow = document.createElement('tr');
+  const columnCount = table.rows[0].cells.length;
+
+  if (tbody.rows.length >= 10) {
+    appendRow.setAttribute('disabled', 'true');
+  }
```

Comment:
The logic for disabling the 'appendRow' button is correct, but there is no logic to re-enable it when the number of rows is less than 10. Consider adding logic to enable the button when rows are removed and the count is below 10.